src/common/models/Projection.ts

```diff
@@ -364,36 +369,47 @@ export class Projection {
   }
 
   private getVaccinationsInfo(
-    actualsTimeseries: Array<ActualsTimeseriesRow | null>,
+    actuals: Actuals,
+    metrics: Metrics,
+    metricsTimeseries: Array<MetricsTimeseriesRow | null>,
   ): VaccinationsInfo | null {
-    if (!DISABLED_VACCINATIONS.includes(this.fips)) {
-      const vaccinatedSeries = actualsTimeseries.map(
-        row => row?.vaccinationsCompleted || null,
-      );
-      const initiatedSeries = actualsTimeseries.map(
-        row => row?.vaccinationsInitiated || null,
-      );
-      const peopleVaccinated = lastValue(vaccinatedSeries),
-        peopleInitiated = lastValue(initiatedSeries);
-      if (peopleVaccinated !== null && peopleInitiated !== null) {
-        const percentCompletedSeries = vaccinatedSeries.map(
-          n => n && n / this.totalPopulation,
-        );
-        const percentInitiatedSeries = initiatedSeries.map(
-          n => n && n / this.totalPopulation,
-        );
-        return {
-          peopleVaccinated,
-          peopleInitiated,
-          percentInitiated: peopleInitiated / this.totalPopulation,
-          percentVaccinated: peopleVaccinated / this.totalPopulation,
-          percentCompletedSeries,
-          percentInitiatedSeries,
-        };
-      }
+    if (DISABLED_VACCINATIONS.includes(this.fips)) {
+      return null;
     }
 
-    return null;
+    const peopleVaccinated = actuals.vaccinationsCompleted;
+    const peopleInitiated = actuals.vaccinationsInitiated;
+    const percentInitiated = metrics.vaccinationsInitiatedRatio;
+    const percentVaccinated = metrics.vaccinationsCompletedRatio;
+
+    if (
+      peopleVaccinated === null ||
+      peopleVaccinated === undefined ||
```

Comment:
There's probably a better way to do this, suggestions included :)

Reply:
I would use `Number.isFinite` instead, it returns `false` for: `NaN`, `-Infinity` and `+Infinity`, `null` and `undefined`

Reply:
ah this doesn't quite work with typescript.  will attempt to do later